lte/gateway/c/oai/tasks/sgw/sgw_task.c

```diff
@@ -77,6 +77,8 @@ static int handle_message(zloop_t* loop, zsock_t* reader, void* arg) {
       sgw_handle_s11_create_session_request(
           spgw_state, &received_message_p->ittiMsg.s11_create_session_request,
           imsi64);
+      clear_protocol_configuration_options(
+          &received_message_p->ittiMsg.s11_create_session_request.pco);
```

Comment:
you should move this to `itti_free_msg_content`.

Reply:
Fixed in latest version